script/ablist.yml

```diff
@@ -11,7 +12,7 @@
     - disabled
 - type: simple
   source: http://ncdj.org/style-guide/
-  note: If possible, describe exacly what this is.
+  note: If possible, describe the specific birth defect.
```

Comment:
@scottgonzalez disabilities don't have to necessarily come from birth defects, right?

Reply:
That's correct, but the only term that will trigger this note (the only `inconsiderate` value) is "birth defect."